lib/synced/strategies/full.rb

```diff
@@ -192,6 +192,15 @@ def search_params
         end]
       end
 
+      def scope_params
+        case @scope
+        when Rental
```

Comment:
i would not include it by default to every model, how many of them would use it? account, website and rental for now, would make it explicitly required for model to be a synced scope. how do you feel about it? maybe with some ArgumentError if model does not respond to `synced_scope_params` on `synchronize` call? just raise if the models don't quack properly :P